### PR TITLE
Css cards

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: yw_website/yw_website/apps/website/templates/includeable/workflow_list.html [hunk 1]
```diff
@@ -21,11 +19,10 @@
 
 {% if workflow_list %}
 
-    <ol id="elementStyle">
-        <script type="text/javascript" src={% static "scripts/home_render_options.js" %}></script>
-
+    <div>
         {% for workflow in workflow_list %}
-            <li>
+        <a href="{% url 'detailed_workflow' workflow.id workflow.version_id %}" style="text-decoration: none; color:inherit;">
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:24]: inline styling

File: yw_website/yw_website/apps/website/templates/includeable/workflow_list.html [hunk 2]
```diff
@@ -36,17 +33,13 @@
...
                     <b>Description: </b>{{ workflow.description }}
                 </div>
 
-                <div style = "text-align:center">
-                    <a class = "more-details-button" href="{% url 'detailed_workflow' workflow.id workflow.version_id %}"> View Details</a>
-                </div>      
+
                 <!-- <button class="details-button" onClick = "description_view()">View Description</button> -->
-            </li>
+            </div class="card">
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:42]: you can omit `class=card` on the closing element

File: yw_website/yw_website/apps/website/templates/includeable/workflow_list.html (hunk 1 above)
Comment (clundin55) [reply yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:24]: For some reason this was not working without the inlining. I would like to merge it for now and deal with it later as we present soon.

File: yw_website/yw_website/apps/website/templates/includeable/workflow_list.html (hunk 2 above)
Comment (clundin55) [reply yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:42]: Gocha

**Review by aniehuser — APPROVED** [review]

**Pull request merged**